test/integration/users.test.js

```diff
@@ -1503,6 +1503,26 @@ describe("Users", function () {
     });
   });
 
+  describe("PATCH /update-archived", function () {
+    it("returns successful response", function (done) {
+      chai
+        .request(app)
+        .patch("/users/update-archived")
+        .set("cookie", `${cookieName}=${superUserAuthToken}`)
+        .end((err, res) => {
+          if (err) {
+            return done(err);
+          }
+
+          expect(res).to.have.status(200);
+          expect(res.body.message).to.equal(
+            "Successfully updated users archived role to true if in_discord role is false"
+          );
```

Comment:
can we also assert on the data field which you have sent in the response to know if its really working or not.